### ISSUE TITLE
[App] Add a "Latest Version" as a version option

### ISSUE THREAD

**Issue by QarthO** [modrinth/code#3095]
### Please confirm the following.

- [x] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate feature requests
- [x] I have checked that this feature request is not on our [roadmap](https://roadmap.modrinth.com)

### Is your suggested feature related to a problem? Please describe.

On the app I would love if there was an option that was basically always latest version. This would primarily be strictly for vanilla and not available for other loaders.
 ![Image](https://github.com/user-attachments/assets/6745fd39-ac45-44fc-b0d9-2d6d01a2a73d)

This is a feature on the vanilla launcher provided by Microsoft and is helpful for people who aren't as Minecraft alliterate and just always want to be on the latest version without having to mess around in settings, (which I believe is the majority of people).

Basically every time you'd open the app it would check and update the instance. Alternatively when you hit play on a instance thats set to latest vanilla, if there's an update there'll be a popup confirmation

"Would u like to update to 1.21.4?"
Then if there are resource/datapacks added there could be a disclaimer explaining that it could cause issues, and maybe an option to clone instance and update the clone

![Image](https://github.com/user-attachments/assets/283c4bc1-6309-40c2-9319-fb79be9788cc)
